cli/src/api/pcluster_api.py

```diff
@@ -47,27 +44,25 @@
         self.stack_name = stack.name
         self.stack_status = stack.status
         self.stack_outputs = stack.outputs
+        if stack.is_working_status:
+            self.head_node_ip = stack.head_node_ip
+            self.user = stack.head_node_user
         self.region = get_region()
         self.version = stack.version
+        self.scheduler = stack.scheduler
 
     def __repr__(self):
         return json.dumps(self.__dict__)
 
 
-class FullClusterInfo(ClusterInfo):
-    """Full representation of a running cluster."""
+class ClusterInfo(ClusterStackInfo):
```

Comment:
Minor: I am not sure about the necessity of this class. Only tow more attributes and no additional method are added upon the base class.

Reply:
Merged with ClusterStackInfo, thanks!

Reply:
Thanks, fixed